tests/target-browser/test-data/target.yaml

```diff
@@ -0,0 +1,12 @@
+apiVersion: triliovault.trilio.io/v1
+kind: Target
+metadata:
+  name: sample-target
+spec:
+  enableBrowsing: true
+  thresholdCapacity: 100Gi
+  nfsCredentials:
+    nfsExport: nfs_server_ip:/src/nfs/targetbrowsertesting/2b3ne
```

Comment:
what's `2b3ne` .. check whether path is correct

Reply:
Sorry This is wrong Fixed correct path